EpiAware/src/models.jl

```diff
@@ -1,10 +1,39 @@
+"""
+    log_infections(y_t, epimodel::EpiModel, latent_process;
```

Comment:
In Julia do you really need to manually write out the arguments of the function in the docs that are defined below? It seems like a bit of duplication that could lead to errors?

Reply:
No, you can use https://github.com/JuliaDocs/DocStringExtensions.jl .